ethcore/src/migrations/account_bloom.rs

```diff
@@ -26,14 +26,26 @@ use util::journaldb;
 use util::{H256, FixedHash, BytesConvertable};
 use util::{Database, DatabaseConfig, DBTransaction, CompactionProfile};
 use std::path::Path;
+use byteorder::{LittleEndian, ByteOrder};
+use std::sync::Arc;
 
-fn check_bloom_exists(db: &Database) -> bool {
+
+fn check_bloom_exists(db: &Arc<Database>) -> bool {
 	let hash_count_entry = db.get(DB_COL_ACCOUNT_BLOOM, ACCOUNT_BLOOM_HASHCOUNT_KEY)
 		.expect("Low-level database error");
 
 	hash_count_entry.is_some()
 }
 
+fn check_space_match(db: &Arc<Database>) -> Result<(), usize> {
+	let db_space = db.get(DB_COL_ACCOUNT_BLOOM, ACCOUNT_BLOOM_SPACE_KEY)
+		.expect("Low-level database error")
+		.map(|val| LittleEndian::read_u64(&val[..]) as usize)
+		.unwrap_or(1048576);
```

Comment:
Well, at least a comment would be helpful. I'm assuming this was the initial size of the bloom, but not 100% sure.